arcgis-ios-sdk-samples/Route & Directions/Route around barriers/README.md

```diff
@@ -1,20 +1,73 @@
+1. Create the route task using `AGSRouteTask.init(url:)` with the URL to a Network Analysis route service.
+2. Get the default route parameters for the service by using `AGSRouteTask.defaultRouteParameters(completion:)`.
+3. When the user adds a stop, add it to the route parameters.
+    i. Normalize the geometry; otherwise the route job would fail if the user included any stops over the 180th degree meridian.
+    ii. Get the name of the stop by counting the existing stops - `stopGraphicsOverlay.graphics.index(of:) + 1`.
+    iii. Create a composite symbol for the stop. This sample uses a blue marker and a text symbol.
+    iv. Create the graphic from the geometry and the symbol.
+    v. Add the graphic to the stops graphics overlay.
+4. When the user adds a barrier, create a polygon barrier and add it to the route parameters.
+    i. Normalize the geometry (see **3i** above).
+    ii. Buffer the geometry to create a larger barrier from the tapped point by calling `class AGSGeometryEngine.bufferGeometry(geometry:byDistance:)`.
+    iii. Create the graphic from the geometry and the symbol.
+    iv. Add the graphic to the barriers overlay.
+5. When ready to find the route, configure the route parameters.
+    i. Set the `returnStops` and `returnDirections` to `true`.
+    ii. Create an `AGSStop` for each graphic in the stops graphics overlay. Add that stop to a list, then call `AGSRouteParameters.setStops(_:)`.
+    iii. Create a `AGSPolygonBarrier` for each graphic in the barriers graphics overlay. Add that barrier to a list, then call `AGSRouteParameters.setPolygonBarriers(_:)`.
+    iv. If the user will accept routes with the stops in any order, set `findBestSequence` to `true` to find the most optimal route.
+    v. If the user has a definite start point, set `AGSRouteParameters.preserveFirstStop` to `true`.
+    vi. If the user has a definite final destination, set `AGSRouteParameters.preserveLastStop` to `true`.
+6. Calculate and display the route.
+    i. Call `AGSRouteTask.solveRoute(with:completion:)` to get an `AGSRouteResult`.
+    ii. Get the first returned route by calling `AGSRouteResult.routes.first()`.
+    iii. Get the geometry from the route as a polyline by accessing the `routeGeometry` property.
+    iv. Create a graphic from the polyline and a simple line symbol.
+    v. Display the steps on the route, available from `directionManeuvers`.
 
+## Relevant API
 
+* AGSDirectionManeuver
+* AGSPolygonBarrier
+* AGSRoute
+* AGSRoute.directionManeuvers
+* AGSRoute.routeGeometry
+* AGSRouteParameters.clearPolygonBarriers
+* AGSRouteParameters.findBestSequence
+* AGSRouteParameters.preserveFirstStop
+* AGSRouteParameters.preserveLastStop
+* AGSRouteParameters.returnDirections
+* AGSRouteParameters.returnStops
+* AGSRouteParameters.setPolygonBarriers
+* AGSRouteResult
+* AGSRouteResult.routes
+* AGSRouteTask
+* AGSStop
+* AGSStop.Name
```

Comment:
This is rendered as a URL on the website.
![image](https://user-images.githubusercontent.com/9660181/84833023-3af82780-afe3-11ea-98d8-4612b402c7b2.png)
Maybe we should inform the docs team to change their rendering patterns?

Reply:
Perhaps, but the symbol is also incorrect. There is no type named `Name` nested under `AGSStop`. This most likely is referring to the `name` property on `AGSStop`, in which case it should be `AGSStop.name`.